config/locales/views/automated_tests/en.yml

```diff
@@ -19,6 +19,10 @@ en:
       in_progress: "in progress"
       problems: "error"
     title: "Automated Testing"
+    display_output:
+      instructors: "Never display"
```

Comment:
I'd like to make these messages much more explicit. Why don't we rename the field `title` to "Display test output to students?" and use the following texts:

- `instructors`: Never display test output to students.
- `instructors_and_students`: Always display test output to students.
- `instructors_and_student_tests`: Only display test output to students for student-run tests.

We can add a note to the documentation that instructors _always_ see test outputs, which is good with me.